### PR TITLE
Social Links need Emphasis

### LINKED ISSUE #12: Social Links need Emphasis
## Description

The links to our community's activity sites are rather lost in the text - I'd suggest using Icons (like the one in my profile) or badges (shield.io or the like) to draw attention to these. If centred, it would also help reduce asymmetric whitespace

### PR DESCRIPTION
**Pull request by krshdev**
fixes #12

### CODE REVIEW

**Review by CodingSpecies — CHANGES_REQUESTED** [review]
~I think you should use HTML tags, so you can change the height of each icon, as they are too big @krishguptadev~

**Review by CodingSpecies — COMMENTED** [review]
Actually, just saw the git diff. The sizes are fine @krishguptadev

File: profile/README.md [hunk 1]
```diff
@@ -18,9 +18,9 @@
...
+    
+<p align="center">
+    <a align="center" href="http://discord.eddiehub.org"><img src="https://camo.githubusercontent.com/c26e422415c93d08e0366cc8f0e8fd43927668a0acac5875038e923a11dd971f/68747470733a2f2f7777772e66726565706e676c6f676f732e636f6d2f75706c6f6164732f646973636f72642d6c6f676f2d706e672f636f6e636f7572732d646973636f72642d6361727465732d766f6575782d666f72746e6974652d6672616e63652d362e706e67" width="50" alt="Join our Discord community here" /></a>
+    <a align="center" href="https://www.youtube.com/user/eddiejaoude"><img src="https://camo.githubusercontent.com/508aaf5f8583bd1a8e080f224b9cde5a017554eae8b5c318d936a6188f3c3c74/68747470733a2f2f7777772e6672656569636f6e73706e672e636f6d2f7468756d62732f796f75747562652d6c6f676f2d706e672f68642d796f75747562652d6c6f676f2d706e672d7472616e73706172656e742d6261636b67726f756e642d32302e706e67" width="50" alt="Subscribe our YouTube channel here" /></a>
```
Comment (CodingSpecies) [inline profile/README.md]: Perhaps @krishguptadev save the icons in an image folder, so the url isn't so long, and is saved on this repo.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (krishguptadev) [reply profile/README.md]: I think it is good there.

Comment (krishguptadev) [reply profile/README.md]: We don't need to add to them to the repo 🤔

Comment (CodingSpecies) [reply profile/README.md]: Ok, I just thought a url like this: `https://camo.githubusercontent.com/508aaf5f8583bd1a8e080f224b9cde5a017554eae8b5c318d936a6188f3c3c74/68747470733a2f2f7777772e6672656569636f6e73706e672e636f6d2f7468756d62732f796f75747562652d6c6f676f2d706e672f68642d796f75747562652d6c6f676f2d706e672d7472616e73706172656e742d6261636b67726f756e642d32302e706e67` was a bit long. Let's see what others say.

**Review by CodingSpecies — COMMENTED** [review]
@krishguptadev is there any updates on this? Thank you.

File: profile/README.md (hunk 1 above)
Comment (krishguptadev) [reply profile/README.md]: We are waiting for other people's opinions on this right? @CodingSpecies

**Review by eddiejaoude — APPROVED** [review]
LGTM

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**